### ISSUE TITLE
typos

### ISSUE THREAD

**Issue by spaette** [JuliaMath/Interpolations.jl#550]
Dispatches
Tangents
analyze
boundary
calculate
coefficients
constructors
determined
inferable
mileage
required
satisfied

```
$ grep -nr Despatches Interpolations.jl
Interpolations.jl/src/iterate.jl:185:# Despatches to getknotindex(ET, iter, idx) if idx is an extrapolated knot,
$ grep -nr Tagents Interpolations.jl
Interpolations.jl/src/monotonic/monotonic.jl:78:Tagents are dertermined at each given point locally,
$ grep -nr analyzy Interpolations.jl
Interpolations.jl/doc/Interpolations_algebra.nb:365:   "To", " ", "analyzy", " ", "bounary", " ", "conditions", " ", "we", " ",
$ grep -nr bounary Interpolations.jl
Interpolations.jl/doc/Interpolations_algebra.nb:365:   "To", " ", "analyzy", " ", "bounary", " ", "conditions", " ", "we", " ",
$ grep -nr calulate Interpolations.jl
Interpolations.jl/src/b-splines/indexing.jl:130:# inner: calulate a single column
$ grep -nr cofficients Interpolations.jl
Interpolations.jl/src/gpu_support.jl:62:This function returns the type of the root cofficients array of an `AbstractInterpolation`.
$ grep -nr copnstructors Interpolations.jl
Interpolations.jl/src/convenience-constructors.jl:1:# convenience copnstructors for constant / linear / cubic spline interpolations
$ grep -nr dertermined Interpolations.jl
Interpolations.jl/src/monotonic/monotonic.jl:78:Tagents are dertermined at each given point locally,
$ grep -nr inferrable Interpolations.jl
Interpolations.jl/src/b-splines/indexing.jl:101:    # TODO: check whether this is inferrable
Interpolations.jl/src/utils.jl:31:# These are not inferrable for mixed-type tuples, so when that's important use `getfirst`
$ grep -nr milage Interpolations.jl
Interpolations.jl/doc/Math.md:60:2. Although the separation of concepts 1-3 from extrapolation behavior is computationally sound, it does allow for some interesting, yet probably nonsensical, combinations of interpolation degrees, boundary conditions and extrapolation behavior. One could imagine for example a constant interpolation (which needs no boundary condition) with linear extrapolation, in which case the interpolating function is a sequence of "steps" with 0-derivative inside the domain, while suddenly having a nonzero derivative outside. Similarly, in most cases where the extrapolation behavior is defined as constant or reflecting, it will make sense to specify matching boundary conditions, but other combinations are entirely supported by `Interpolations.jl`; your milage may vary, very much...
$ grep -nr requred Interpolations.jl
Interpolations.jl/doc/Interpolations.jl.ipynb:99:    "All interpolations of quadratic degree or higher require a prefiltering step, which entails solving a tridiagonal system of equations (details can be found for example in [this paper](http://dx.doi.org/10.1109/42.875199)), in order to make the interpolating function pass through the data points. `Interpolations.jl` takes care of solving this system for you, but in order to close the system a boundary condition is requred."
$ grep -nr satisifed Interpolations.jl
Interpolations.jl/doc/Interpolations_algebra.nb:156:conditions are satisifed by the following cubic polynomials (TODO: show \
$
```

If there are any modifications indicated for the below shell script please inform.

```
$ cat typos.sh
#!/bin/sh

sed -i "s/Despatches/Dispatches/g" Interpolations.jl/src/iterate.jl
sed -i "s/Tagents/Tangents/g" Interpolations.jl/src/monotonic/monotonic.jl
sed -i "s/analyzy/analyze/g" Interpolations.jl/doc/Interpolations_algebra.nb
sed -i "s/bounary/boundary/g" Interpolations.jl/doc/Interpolations_algebra.nb
sed -i "s/calulate/calculate/g" Interpolations.jl/src/b-splines/indexing.jl
sed -i "s/cofficients/coefficients/g" Interpolations.jl/src/gpu_support.jl
sed -i "s/copnstructors/constructors/g" Interpolations.jl/src/convenience-constructors.jl
sed -i "s/dertermined/determined/g" Interpolations.jl/src/monotonic/monotonic.jl
sed -i "s/inferrable/inferable/g" Interpolations.jl/src/b-splines/indexing.jl
sed -i "s/inferrable/inferable/g" Interpolations.jl/src/utils.jl
sed -i "s/milage/mileage/g" Interpolations.jl/doc/Math.md
sed -i "s/requred/required/g" Interpolations.jl/doc/Interpolations.jl.ipynb
sed -i "s/satisifed/satisfied/g" Interpolations.jl/doc/Interpolations_algebra.nb
$
```

**Comment by mkitti** [comment]
Thanks. Could you submit a pull request?

**Comment by spaette** [comment]
PR submitted.

**Comment by mkitti** [comment]
Merged in #551. Thanks.
